tests/characteristicsTypes_test.py

```diff
@@ -48,5 +48,4 @@ def test_get_short(self):
         self.assertEqual(CharacteristicsTypes.get_short(CharacteristicsTypes.DOOR_STATE_TARGET), 'door-state.target')
         self.assertEqual(CharacteristicsTypes.get_short(CharacteristicsTypes.AIR_PURIFIER_STATE_CURRENT),
                          'air-purifier.state.current')
-        #self.assertEqual(CharacteristicsTypes.get_short('1a'), 'Unknown Characteristic 1a')
-        self.assertEqual(CharacteristicsTypes.get_short('1a'), 'lock-management.auto-secure-timeout')
+        self.assertEqual(CharacteristicsTypes.get_short('1a'), 'Unknown Characteristic 1a')
```

Comment:
With the comment on `get_short` this would again return `lock-management.auto-secure-timeout`

Reply:
Yes, if .upper() is left inside `get_short` then it would.
In my PR i removed it from `get_short`, that's why i returned it back to 'Unknown'.

So just need to agree if it is enough to normalize UUID's inside controller.py
or also there is need to normalize them inside every lookup method?